services/node-services/src/cancel_test.ts

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2023 - Restate Software, Inc., Restate GmbH
+//
+// This file is part of the Restate e2e tests,
+// which are released under the MIT license.
+//
+// You can find a copy of the license in file LICENSE in the root
+// directory of this repository or package, or at
+// https://github.com/restatedev/e2e/blob/main/LICENSE
+
+import * as restate from "@restatedev/restate-sdk";
+
+import {
+    CancelTestService as ICancelTestService,
+    BlockingService as IBlockingService,
+    protobufPackage,
+    BlockingServiceClientImpl,
+    Response,
+    BlockingOperation,
+    BlockingRequest,
+} from "./generated/cancel_test";
+import { Empty } from "./generated/google/protobuf/empty";
+import { AwakeableHolderServiceClientImpl } from "./generated/awakeable_holder";
+
+export const CancelTestServiceFQN = protobufPackage + ".CancelTestService";
+export const BlockingServiceFQN = protobufPackage + ".BlockingService";
+
+export class CancelTestService implements ICancelTestService {
+    async verifyTest(request: Empty): Promise<Response> {
+        const ctx = restate.useContext(this);
+        const isCanceled = await ctx.get<boolean>("canceled") ?? false;
+
+        return Response.create({ isCanceled: isCanceled });
```

Comment:
Don't need to wrap it in `Response.create`, it's type safe anyway.

Reply:
Will remove it.